src/Package/Impl/DataInspect/VariableView.xaml

```diff
@@ -160,12 +154,55 @@
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
 
-        <ComboBox x:Name="EnvironmentComboBox" SelectionChanged="EnvironmentComboBox_SelectionChanged" Margin="4">
+        <ComboBox x:Name="EnvironmentComboBox"
+                  Margin="4"
+                  ItemsSource="{Binding Environments}"
+                  SelectedItem="{Binding SelectedEnvironment}"
+                  SelectionChanged="EnvironmentComboBox_SelectionChanged">
             <ComboBox.ItemTemplate>
                 <DataTemplate>
-                    <TextBlock Text="{Binding Name}" />
+                    <StackPanel Orientation="Horizontal">
+                        <imaging:CrispImage Width="16" Height="16">
+                            <imaging:CrispImage.Style>
+                                <Style TargetType="{x:Type imaging:CrispImage}">
+                                    <Setter Property="Moniker" Value="{x:Static imagecatalog:KnownMonikers.ObjectPublic}"/>
```

Comment:
If lambda can handle `switch` - sure

---
In reply to: [63223696](https://github.com/Microsoft/RTVS/pull/1770#discussion_r63223696) [](ancestors = 63223696)